Change order of method generation

delocalizer opened the pull request:
Move `generateMethods` and `generateMethodsForSuperInterfaces` to be _after_ generation of the RdfId accessors, so those preferentially get made as the versions that forward to the supportsId field accessors, in particular the setter implementation that throws `Exception` when id exists already.

Previously, since everything that makes it past  [L99](https://github.com/mhgrove/Empire/compare/develop...delocalizer:master#diff-a3e0020fff3b9629a72bdfa9b4a38f93L99) implements `SupportsRdfId`, the `generateMethodsForSuperInterfaces` at   [L146](https://github.com/mhgrove/Empire/compare/develop...delocalizer:master#diff-a3e0020fff3b9629a72bdfa9b4a38f93L146) would create "naive" setter first and the test at [L155](https://github.com/mhgrove/Empire/compare/develop...delocalizer:master#diff-a3e0020fff3b9629a72bdfa9b4a38f93L155) will never be true.

delocalizer committed:
Merge pull request #1 from mhgrove/master

merge back

delocalizer committed:
* Change order of method creation in InstanceGenerator, so that RdfId
  accessors are the ones that forward to SupportsRdfIdImpl field

delocalizer committed:
* changes from mhgrove
Merge branch 'develop'

mhgrove reviewed (changes requested):
Makes sense. Can you add a test case for this?

delocalizer committed:
test for https://github.com/mhgrove/Empire/pull/115

delocalizer committed:
* ignore Eclipse files

delocalizer commented:
Hi Mike;
I added a test. Forgot that I'd also added changes to .gitignore to ignore my IDE artifacts before pushing so I'll close this, make a new branch and PR with just the relevant things
